src/configLoader.js

```diff
@@ -13,12 +14,20 @@ export function configLoader(configPath) {
         return process.exit(1);
     }
 
+    const rawConfig = fs.readFileSync(configPath, 'utf8');
+    const compiledConfig = rawConfig.replace(/\$\{([A-Za-z_]+)\}\$/g, (match, variableName) => {
```

Comment:
Does this regex need the extra `$` at the end? Right now this will match `${ENV}$` not `${ENV}`

Reply:
yes, you're correct, i didn't actually test this version. thanks for catching it, it is now fixed